docker/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Version: 0.0.1
+FROM ubuntu:19.04
+
+ENV REFRESHED_AT 2018-12-14
+ENV container docker
+
+RUN apt-get -y update; apt-get clean all
+RUN apt-get -y install cmake
+RUN apt-get -y install default-jdk
+RUN apt-get -y install yui-compressor 2.4.7
```

Comment:
You should do all or at least most `RUN` steps in one step to keep the container file size down

https://developers.redhat.com/blog/2016/03/09/more-about-docker-images-size/